datasetinsights/evaluation_metrics/average_quaternion_error.py

```diff
@@ -0,0 +1,50 @@
+r"""Average Quaternion error metric.
+"""
+
+import numpy as np
+
+from .base import EvaluationMetric
+from .exceptions import NoSampleError
+
+
+class AverageQuaternionError(EvaluationMetric):
```

Comment:
Can you add a unittest for this? Here is an [example](https://github.com/Unity-Technologies/datasetinsights/blob/master/tests/evaluation_metrics/test_average_relative_error.py).